lib/runner.js

```diff
@@ -22,11 +22,16 @@ function runner(options, next) {
 
     /*jslint unparam:true*/
     exec(args.join(' ').trim(), function (err, stdout, stderr) {
-
       return parseOutput(stderr, next);
+    }).on('exit', function (exitCode) {
+      process.exitCode = exitCode;
     });
     /*jslint unparam:false*/
   });
 }
 
+process.on('exit', function () {
+  process.exit(process.exitCode);
```

Comment:
@agirorn I suggest that your remove this now as node.js 0.10 is long out of support, The we should look at getting this landed